### ISSUE TITLE
[DESIGN] Rethink the Close/Removal flow for batch change

### ISSUE THREAD

**Issue by danielmarquespt** [sourcegraph/sourcegraph#38713]

**Comment by courier-new** [comment]
This is a symptom of the server-side workflow being completely separated from the existing src-cli batch change stuff. The "right" solution is unifying the two workflows, but as a stop-gap I wonder if we could just add "Close" to the actions menu we show from server-side screens when there's more than one action you can take on that screen:

![Image](https://user-images.githubusercontent.com/8942601/180279285-1dfb580e-a94a-4035-92ae-0cc95052fb0f.png)

**Comment by chrispine** [comment]
Assuming this is the design (not implementation) represented in this issue, I don't have a strong sense of the relative priority of this compared to our other design tasks

**Comment by danielmarquespt** [comment]
> Assuming this is the design (not implementation) represented in this issue, I don't have a strong sense of the relative priority of this compared to our other design tasks

Yes, this is just design, because I think it required and deeper re-think of the flow. Given our priorities right now, I think this is blacklog, but we can discuss in Sprint Planning

**Comment by LawnGnome** [comment]
I guess the question is whether we want to add the Close action as a temporary stopgap, or whether that's not going to move the needle enough to worth the effort. (My estimate on that is 0.5d, but low confidence, as I have minimal knowledge of that component.)

**Comment by courier-new** [comment]
Update: we implemented the stop-gap as its own ticket: https://github.com/sourcegraph/sourcegraph/issues/39446. This ticket now exclusively pertains to the longterm better design for this.

I think it's worth backlogging this in favor of higher priority design work.

**Comment by courier-new** [comment]
Backlogging this for now. It isn't high priority in light of other strategic-ready design efforts.
